src/cpp/desktop/DesktopWebPage.cpp

```diff
@@ -358,7 +346,8 @@ void initSafeHosts()
    safeHosts_ = {
       ".youtube.com",
       ".vimeo.com",
-      ".c9.ms"
+      ".c9.ms",
+      ".marketo.com"
```

Comment:
Don't think we need to add this to safe hosts; the issue wasn't that this wasn't being allowed (which is also the case in RStudio 1.2 and 1.3 and didn't seem to be a concern) but that we are now opening it in system browser.